jdaviz/core/custom_units.py

```diff
@@ -6,6 +6,13 @@
 PIX2 = u.pix * u.pix
 
 
+# Add spaxel to enabled units
+def enable_spaxel_unit():
```

Comment:
we should probably define the PIX2 unit like this right? should i make a ticket?